cfme/cloud/provider/openstack.py

```diff
@@ -83,9 +88,11 @@ def from_config(cls, prov_config, prov_key, appliance=None):
                    hostname=prov_config['hostname'],
                    ip_address=prov_config['ipaddress'],
                    api_port=prov_config['port'],
+                   api_version=prov_config.get('api_version', 'Keystone v2'),
```

Comment:
if api_version is None, this won't default to 'Keystone v2', it will be None. I think you probably want to default to 'Keystone v2' when api_version is None, not only when api_version isn't set.

Reply:
Got it, done.